projects/packages/forms/src/blocks/shared/hooks/use-get-inner-block.js

```diff
@@ -0,0 +1,19 @@
+import { store as blockEditorStore } from '@wordpress/block-editor';
+import { useSelect } from '@wordpress/data';
+
+export default function useGetInnerBlock( clientId, blockName ) {
```

Comment:
There's a path to combine this with [useSibling](https://github.com/Automattic/jetpack/pull/43327/files#diff-69c941a5982946d2b26ef728f14dba96e841e0b1d2a88746cdf73841c36f82abR23), but I left it in case we roll back these changes.

Reply:
The latest code seems to be going back to the old approach of setting css values on field blocks, which I'd moved away from, so I'll have a look at if we can remove it.

I think we need to have every field working the same, as otherwise it'll be too challenging to keep up with all the inconsistencies.

Reply:
Ah yeah, fair enough. Wasn't thinking. I'll see if they can be added to the options block itself.

Reply:
Done in: https://github.com/Automattic/jetpack/pull/43327/files/#diff-b8bb9a578de10b62654dcd799ee2ef0bcac22153d1d2537d43a6df2bca86ec79R17

They're doubled up by necessity (on label and options blocks) since the options needs to calculate inner padding. Same as inputs I see. I think my brain was trying to avoid the duplication. I dunno 😄 

**TODO** on Monday: ensure the CSS vars are applied to radio/checkboxes on the frontend. (They're not)

Reply:
> Same as inputs I see. I think my brain was trying to avoid the duplication.

My thinking is it still ends up being duplication whether on the field or input/options blocks, but at least on the input/options blocks the attributes we need are the ones from the block itself, so it saves a step.